scripts/archiving.cake

```diff
@@ -3,14 +3,19 @@
 
 using System.IO.Compression;
 
+string GetPackagePrefix(string project)
+{
+    return project.EndsWith(".Stdio") ? string.Empty : "." + project.Substring(project.IndexOf('.')+1).ToLower();
```

Comment:
Did you mean this?

```C#
return project.EndsWith(".Stdio") ? string.Empty : project.Substring(project.IndexOf('.')).ToLower();
```

Reply:
The extra '.' was intentional so that the package comes out like `omnisharp.http-win-x64` vs `omnisharphttp-win-x64`.

Reply:
Sure, I get that. But you already have a "." because you're searching for ".". You should be able to just use `project.IndexOf('.')` rather than `project.IndexOf('.')+1`.

Reply:
stop making sense...